### DISCUSSION TITLE
"Zurückblicken" mit Track IR

### DISCUSSION THREAD

**Discussion by Z0dd3L** [Mogli12/KeyboardSteer#759] [Q&A]
Hallo,
mir ist aufgefallen bei der Funktion "Zurückblicken(innen)" in Verbindung mit Track IR setzt die Kamera bei Rückwärtsfahrt vor die Windschutzscheibe quasi auf die Motorhaube (Blickrichtung nach hinten). In dieser Ansicht ist sowohl die Kabine und auch die Spielfigur zu sehen.

Ohne Track IR funktioniert es wie gewollt.

**Comment by Mogli12** [comment]
Das kann sein. Ich denke, dass man entweder Track IR oder die Kamerarotation von VCA verwendet.
